client/state/ui/language/reducer.js

```diff
@@ -24,4 +25,34 @@ export const localeSlug = createReducer(
 	localeSlugSchema
 );
 
-export default combineReducers( { localeSlug } );
+/**
+ * Tracks the state of the ui locale
+ *
+ * @param  {Object} state  Current state
+ * @param  {Object} action Action payload
+ * @return {Object}        Updated state
+ *
+ */
+export const isRtl = createReducer(
+	false,
+	{
+		[ LOCALE_SET ]: ( state, action ) => {
+			const localeSlugFromAction = action.localeSlug;
+
+			if ( ! localeSlugFromAction ) {
+				return state;
```

Comment:
Here you are changing the behavior:
before:
- if `localeSlug` is invalid, `isRtl` returns `null` (which means "unknown")
- the selector returns the `null` value if it's there

after:
- if `localeSlug` is invalid, the previous value of `isRtl` is left there unchanged
- the selector will change `null` to `false`

Are you sure you wanted to do the change?

Reply:
Hmm, initially, I thought changing it would be okay. But let's not modify stuff here.